README.md

```diff
@@ -33,3 +33,67 @@ The `git clonefork` can be used by running `git clonefork {REMOTE_ORIGIN_URL}` w
 ## Installation
 To install the utils, use the `./install.sh` script provided in this repo. If you would like to specify which util to install rather than install all of them, you can
 use `./install.sh -n <name_of_util`. For example, to install `git-sync` only, you could run `./install.sh -n git-sync`.
+
+
+## How to set up an SSH Key with GitHub
+
+To set up an SSH key with GitHub, follow these steps:
```

Comment:
Instead of this long (really good) description of how to set up an SSH key with GitHub, can you instead provide links to both the GitHub documentation and the GitLab documentation for how to do this? We don't want to have to maintain this documentation if something ever changes on one of those platforms